src/Synth/ADnote.cpp

```diff
@@ -1936,6 +1936,9 @@ void ADnote::releasekey()
     NoteGlobalPar.FreqEnvelope->releasekey();
     NoteGlobalPar.FilterEnvelope->releasekey();
     NoteGlobalPar.AmpEnvelope->releasekey();
+    NoteGlobalPar.FreqLfo->releasekey();
+    NoteGlobalPar.FilterLfo->releasekey();
+    NoteGlobalPar.AmpLfo->releasekey();
```

Comment:
I think this needs to be added for PAD and possibly for SUB. For the effects, we don't support fadein/fadeout, so it's just PAD and SUB?

Reply:
SUB has no LFOs. I add it to PAD